src/apps/monitoring/iplabel/newtest/restapi/mode/scenarios.pm

```diff
@@ -299,6 +304,10 @@ Filter robots (can be a regexp).
 
 Filter scenarios (can be a regexp).
 
+=item B<--timeframe>
+
+Define timeframe in seconds (by default it's the time between two executions).
```

Comment:
We should add timeframe default value here

`Define timeframe (time between two executions) in seconds (default value: 300 ).`

Reply:
Is this default value explanation wrong or anything else ?